.github/workflows/ci-tests.yml

```diff
@@ -22,11 +22,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - run: npm ci
       - run: npm run build
-      - run: npm test
```

Comment:
Will tests still be run? Before this change we run "npm test" on all the packages.

If yes, then it's all good 🚀

Reply:
Yes `npm test` wil run just as before and `npm run coverage` will run the tests **and** make the coverage reports. 

(I temporary changed one of the tests just to make sure ;-) )